crt_portal/static/sass/custom/landing.scss

```diff
@@ -169,6 +169,9 @@ $arrow-height: 24px;
       background: color($theme-color-warning);
       color: black !important;
     }
+    a, a:visited {
+      color: white;
```

Comment:
curious why this change?

Reply:
For the mobile background, to match the rest of the text which is white.